### DISCUSSION TITLE
Strange behavior on "Balance Power" showing load from devices on non monitored circuits...

### DISCUSSION THREAD

**Discussion by Infernoken76** [emporia-vue-local/esphome#158] [General]
I'm starting to believe the "Balance Power" implementation is unusable...

This is the following I'm seeing:
![image](https://user-images.githubusercontent.com/128485046/234104253-a94e0845-3873-4821-8a3e-a08f1551236f.png)

~486W of load on the "Balance Power" while the actual 7 circuits connected and monitored are not consuming anything...
(And I commented out the non connected 8-16 circuits, so these do not contribute to the "Balance Power" reading.

But when looking at the "Total Power" it indeed shows ~510W of consumption, due to other devices in my home consuming power.

So how can the "Balance Power" that is looking at individual circuits, show the consumption of other devices that are on different breakers that are not monitored ???

Something does not add up here...

**Comment by flaviut** [comment]
The calculation is correct:

![image](https://user-images.githubusercontent.com/5213469/234120437-4d44a1ab-97c2-4c64-8cc2-fb23c882bd6c.png)

I think you've misunderstanding what balance power means. It's the total power that's not counted by any of the circuit clamps.

**Reply by Infernoken76** [reply]
> The calculation is correct:
> 
> ![image](https://user-images.githubusercontent.com/5213469/234120437-4d44a1ab-97c2-4c64-8cc2-fb23c882bd6c.png)
> 
> I think you've misunderstanding what balance power means. It's the total power that's not counted by any of the circuit clamps.

Ok, you got me…
Completely misunderstood the meaning.
I’ll give it another name in my setup like “Other Power” to make it more understandable…

Thanks for all the insights, now near to a perfect monitoring…
